src/core/services/gesture/gesture.js

```diff
@@ -248,7 +248,7 @@ function MdGesture($$MdGestureHandler, $$rAF, $timeout) {
         if (touchActionProperty) {
           // We check for horizontal to be false, because otherwise we would overwrite the default opts.
           this.oldTouchAction = element[0].style[touchActionProperty];
-          element[0].style[touchActionProperty] = options.horizontal === false ? 'pan-y' : 'pan-x';
+          element[0].style[touchActionProperty] = options.horizontal ? 'pan-y' : 'pan-x';
```

Comment:
Cause it's the other way around:

 | Value | Description |
| ------ | --- |
| pan-x | The browser is only allowed to perform the horizontal scroll default action. |
|  pan-y | The browser is only allowed to perform the vertical scroll default action. |

There for we need

| State | Value |
| ----- | ------ |
| horizontal | pan-y |
| vertical |  pan-x |